pkg/querier/querier.go

```diff
@@ -27,6 +27,9 @@ type Config struct {
 	MaxSamples               int
 	IngesterMaxQueryLookback time.Duration
 
+	// IngesterMetricMinAge minimum time between a metric being ingested and saved to the store.
+	IngesterMetricMinAge time.Duration `yaml:"ingester_metric_min_age"`
```

Comment:
I find this config option name difficult to understand (ie. it's called "min age" but looks more a "max age for queries to be sent to ingesters only". Do you have any better option for the name?

Reply:
Yea I tried to keep it similar to the option I moved, `MinChunkAge` so it refers to the minimum age of a metric before sending to the store. But I don't really like the name either and would love a better idea.

Reply:
To me, this new option is similar to `IngesterMaxQueryLookback`, in the sense it does the same thing but applied to the store. A better naming may be `store_min_query_lookback` (`-querier.store-min-query-lookback`), in the sense it's the minimum query lookback time required to hit the storage.

Reply:
I like that name much better. Changed.